common/web/gesture-recognizer/src/test/auto/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+/*
+ * VS Code Intellisense needs this helper in order to properly use subpath imports in the
+ * test specs found under the `headless` subfolder.  Otherwise, it'll report errors while
+ * editing - even if the tests themselves actually work.
+ */
+{
+  // Enabling the following line re-breaks it for some reason?
+  "extends": "../../../../tsconfig.kmw-main-base.json",
```

Comment:
:+1:

Reply:
Right... its old form before the fix you suggested certainly did.  I'll remove the comment now that it no longer applies.